tests/e2e/e2e_setup_test.go

```diff
@@ -55,7 +56,13 @@ type chainConfig struct {
 
 const (
 	// osmosis version being upgraded to (folder must exist here https://github.com/osmosis-labs/osmosis/tree/main/app/upgrades)
-	upgradeVersion = "v9"
+	upgradeVersion string = "v10"
+	// is this version a fork
+	isFork bool = true
+	// if isFork is true, this is the forkHeight
+	forkHeight int = 4713065
+	// if not skipping upgrade, how many blocks we allow for fork to run pre upgrade state creation
+	forkHeightOffset int = 60
```

Comment:
I know it's verbose but I'd prefer verbosity for clarity here. I was wondering if that's what the offset was for before seeing the comment
```suggestion
	forkHeightPreUpgradeOffset int = 60
```